noetic/distribution.yaml

```diff
@@ -3208,6 +3208,16 @@ repositories:
       url: https://github.com/ros-perception/image_transport_plugins.git
       version: noetic-devel
     status: unmaintained
+  imagezero_transport:
+    release:
+      packages:
+      - imagezero
+      - imagezero_image_transport
+      - imagezero_ros
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/swri-robotics-gbp/imagezero_transport-release.git
+      version: 0.2.5-1
```

Comment:
@danthony06 no `source` nor `doc` entries?

Reply:
We've previously released this package for several iterations of ROS. Do we need to start providing these?

Reply:
It's not required, though we do recommend it.  I'm going to go ahead and merge this in, but encourage you to consider adding a doc and source entry.